cli/src/pcluster/constants.py

```diff
@@ -206,3 +207,36 @@
 
 IAM_NAME_PREFIX_LENGTH_LIMIT = 30
 IAM_PATH_LENGTH_LIMIT = 512
+
+
+# Features support
+# By default, all features are considered supported.
+# To mark a feature as unsupported for certain regions,
+# add the entry to the map below by region prefixes.
+class Feature(Enum):
+    """
+    Enumeration of features.
+
+    We do not expect this enumeration to list all the features,
+    but at least those that are considered for feature flagging.
+    """
+
+    BATCH = "AWS Batch"
+    DCV = "Nice DCV"
+    FSX_LUSTRE = "FSx Lustre"
+    FSX_ONTAP = "FSx ONTAP"
+    FSX_OPENZFS = "FSx OpenZfs"
+
+
+UNSUPPORTED_FEATURES_MAP = {
```

Comment:
what about having this map by feature, rather than by region?

```python
UNSUPPORTED_FEATURES_MAP = {
  Feature.BATCH: ["ap-northeast-3", "us-iso"]
  Feature.DCV: [...]
```

Reply:
This corresponds that the validator is for the feature. We are registering the validator when the feature is enabled in the config, so the feature is our starting point to see the limitations.

Reply:
I like it, will do